core/frontend/src/Marker.ts

```diff
@@ -335,6 +347,8 @@ export abstract class MarkerSet<T extends Marker> {
   public minimumClusterSize = 1;
   /** The set of Markers in this MarkerSet. Add your [[Marker]]s into this. */
   public get markers(): Set<T> { return this._markers; }
+  /** The radius (in pixels) for clustering markers, default 0 which implies calculating the radius based on the visible marker imageSize/size. */
```

Comment:
Clarify documentation: the imageSize/size behavior is applied for any value <= 0.

Reply:
Done